### ISSUE TITLE
onesync infinity not working

### ISSUE THREAD

**Issue by henrikmaa** [plunkettscott/interact-sound#20]
When trying to use onesync infinity all players can here sound when its played

`TriggerServerEvent('InteractSound_SV:PlayWithinDistance', 4.0, 'demo', 0.1)`

Also tried to remove the if statement to use original or onesync to only

```
RegisterServerEvent('InteractSound_SV:PlayWithinDistance')
AddEventHandler('InteractSound_SV:PlayWithinDistance', function(maxDistance, soundFile, soundVolume)

TriggerClientEvent('InteractSound_CL:PlayWithinDistanceOS', -1, GetEntityCoords(GetPlayerPed(source)), maxDistance, soundFile, soundVolume)
end)
```

**Comment by LordXenaroth** [comment]
Yeah same issue here, did you tried something else?

**Comment by JSneaux** [comment]
Did you figure this out? Still trying to find a fix my side of things too, to no avail.

**Comment by Tinky124** [comment]
Fixed #21 #22  :)

**Comment by plunkettscott** [comment]
OneSync is fixed in release 0.5. Feel free to reopen this if your problem is not resolved.

**Comment by Legacy-TacticalGamingInteractive** [comment]
The issue seems to still be happening
